src/globalStyles.ts

```diff
@@ -25,4 +25,14 @@ export const globalStyles = css`
   .rt-SelectItem > span:not(.rt-SelectItemIndicator) {
     width: 100%;
   }
+
+  /* Make Monaco scroll thumbs rounder */
```

Comment:
I tried matching the track styles as well, but the vertical and horizontal scrollbars would overlap each other in the bottom right corner and it looked off. Maybe we'll find a solution in future